botocore/exceptions.py

```diff
@@ -814,3 +814,18 @@ class EndpointResolutionError(EndpointProviderError):
 
 class UnknownEndpointResolutionBuiltInName(EndpointProviderError):
     fmt = 'Unknown builtin variable name: {name}'
+
+
+class InvalidRequestMinCompressionSizeError(BotoCoreError):
+    fmt = (
+        'Invalid value provided to "request_min_compression_size_bytes": '
+        '{input_value} must be an integer greater than or equal to 0 and '
+        'less than or equal to 10485760.'
```

Comment:
Can we move the 10485760 into a constant so that we can be sure to always use the same number in the check and in the exception message? How do other exception messages in this file deal with similar constant values?

Reply:
Arguably this value shouldn't even be hardcoded. We can format the min/max in the exception.